core/test/gather/base-gatherer-test.js

```diff
@@ -1,85 +0,0 @@
-/**
- * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
- */
-
-import Gatherer from '../../gather/base-gatherer.js';
-import {fnAny} from '../test-utils.js';
-
-/** @type {any} */
-const fakeParam = {};
-
-describe('BaseGatherer', () => {
-  it('should fullfill the contract of both interfaces', async () => {
-    const gatherer = new Gatherer();
-
-    // Fraggle Rock Gatherer contract
-    expect(typeof gatherer.meta).toBe('object');
-    expect(gatherer.getArtifact(fakeParam)).toBe(undefined);
-
-    // Legacy Gatherer contract
-    expect(typeof gatherer.name).toBe('string');
-    expect(await gatherer.beforePass(fakeParam)).toBe(undefined);
-    expect(await gatherer.pass(fakeParam)).toBe(undefined);
-    expect(await gatherer.afterPass(fakeParam, fakeParam)).toBe(undefined);
-  });
```

Comment:
After removing the legacy `beforePass`/`afterPass` stuff this test file really isn't testing anything. The base gatherer gets plenty of test coverage from other files.